StyleGuide.js

```diff
@@ -0,0 +1,4 @@
+// Javascript Hooks
+//  When hooking onto DOM elements, decorate your HTML with `js-{name}`. This means you are separating your CSS from
```

Comment:
make this clearer that youre talking about class names "decorate" isnt descriptive enough